models/classes/Lists/Business/Service/ValueCollectionService.php

```diff
@@ -91,4 +97,19 @@ public function persist(ValueCollection $valueCollection): bool
 
         return false;
     }
+
+    private function setUserDataLanguage(ValueCollectionSearchRequest $searchRequest): void
+    {
+        $user = $this->getServiceLocator()->get(SessionService::class)->getCurrentUser();
```

Comment:
```suggestion
        /** @var SessionService $sessionService **/
        $sessionService = $this->getServiceLocator()->get(SessionService::class);

        $user = $sessionService->getCurrentUser();
```

Minor change just to help us to track usage of method `getCurrentUser` using the IDE